src/cowboy_http2.erl

```diff
@@ -1137,6 +1157,21 @@ reset_rate(State0=#state{reset_rate_num=Num0, reset_rate_time=Time}) ->
 			{ok, State0#state{reset_rate_num=Num}}
 	end.
 
+cancel_rate(State0=#state{cancel_rate_num=Num0, cancel_rate_time=Time}) ->
```

Comment:
This function should be just below the function that calls it (line 595).

Reply:
Done. I renamed it to `cancel_rate_limit` and simplified it a bit by just returning the new state (not wrapped in an ok-tuple) and calling `terminate` directly instead of returning `error`.